Sources/TuistKit/Commands/Inspect/InspectBuildCommand.swift

```diff
@@ -0,0 +1,26 @@
+import ArgumentParser
+import Foundation
+
+struct InspectBuildCommand: AsyncParsableCommand {
```

Comment:
Naming is hard 😬. Is `inspect build` the right name? When I think of inspect, I don't think of pushing a report to a server. Should we name it `tuist xcode report-activity-logs`?

Reply:
> Naming is hard 😬. Is inspect build the right name? When I think of inspect, I don't think of pushing a report to a server. Should we name it tuist xcode report-activity-logs?

The goal is that this command when running in an interactive environment would actually show the build in our dashboard as explained here: https://github.com/tuist/tuist/pull/7340#discussion_r1975275246

I would rather then not have two different commands, doing almost the same thing: `tuist xcode report-activity-logs` and `tuist inspect build`. wdyt?

Reply:
I see your point. My concern is about the semantics of "inspect". When I think of "inspect" I imagine something being inspected here in my machine. I find that verbs like "track" "report" or "publish" overlap more with that the command does. Opening the browser can be a side effect:

```bash
tuist publish/track/report --open-browser # Defaults to last build  but down the line you can make that controllable with something like -n 5
```

It might just be me though.

Reply:
When we get to a point where we open the browser with the current build, `inspect` will make a lot more sense.

I'd also like not to add an extra top-level command unless we'll be adding more. So, as of now, I agree, but going forward, `tuist inspect build` will make more sense. We'll need to make the same decision for tracking module size with Rosalind. Will it be:
- `tuist inspect artifact App.app` where on CI, we'd only publish the artifact to the server, possibly printing out some metadata to the stdout?
- or `tuist inspect artifact App.app` for local _and_ `tuist report artifact App.app` which would be mostly used on the CI?

I'm leaning to the former, but curious to know your thoughts.

Reply:
We decided to keep `tuist inspect build` given our long-term plan to open the build log in the browser in interactive sessions.